etc/fixtures/add_user.js

```diff
@@ -0,0 +1,75 @@
+const usage = "\n\nmongosh --eval \"var help=false; [options]\" /tmp/fixtures/add_user.js\n" +
+    "\nOptions:\n" +
+    "    help             if true or undefined, print this help message\n" +
+    "    userFile         default:  /tmp/fixtures/example-adding-users.json    file to read for new users\n" +
+    "    host             default:  localhost    host to use in the URI for connecting to mongo\n" +
+    "    port             default:  27017    port to use in the URI for connecting to mongo\n" +
+    "    databaseName     default:  rosalution_db    databaseName to use\n" +
+    "\nExample: mongosh --eval \"var help=false; userFile='/tmp/fixtures/example.json', host='127.0.0.1'," +
+    " port='27017', databaseName='your_db_name'\" /tmp/fixtures/add_user.js\n";
+
+if (typeof help === 'undefined' || help !== false) {
```

Comment:
As discussed, we want usage to print out only if there is a failure with the usage or if it's specifically requested by the user. 

The script should be able to run with default values and no extra variables set and not print out usage.